src/status_im2/contexts/chat/messages/message/deleted_message/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im2.contexts.chat.messages.message.deleted-message.view
```

Comment:
could you please rollback this file, see https://github.com/status-im/status-mobile/pull/14573/files#diff-f19099d11cab857c4ab6061a65d7054b139f4e5feebd3d01860382f554cdd7c4

Reply:
done